test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -4066,7 +4013,7 @@
                     "System.Threading.Tasks": "4.3.0"
                 }
             },
-            "System.Threading.Overlapped/4.3.0": {
+            "System.Threading.Overlapped/4.0.1": {
```

Comment:
This and `System.Net.Sockets` appear to be downgrades, which is interesting. But they're part of the framework so I believe this will be okay.

Reply:
This is potentially impactful. What is the unification policy? If a customer was on 4.3.0 before, they would get the unified behavior, but that would change with this update.